app/models/game.rb

```diff
@@ -7,5 +7,19 @@ class Game < ActiveRecord::Base
   
   has_many :pieces
   has_many :players
+
+  def piece_present?(x, y)
+    if self.pieces.where(x_position: x, y_position: y).any?
```

Comment:
You can simply return the result of this method - `.any?` returns a boolean, no need to return a different boolean